jerry-core/parser/js/js-parser-statm.c

```diff
@@ -334,9 +334,17 @@ parser_parse_enclosed_expr (parser_context_t *context_p) /**< context */
  * Parse var statement.
  */
 static void
-parser_parse_var_statement (parser_context_t *context_p) /**< context */
+parser_parse_declaration_statement (parser_context_t *context_p) /**< context */
```

Comment:
The original `parser_parse_var_statement` contains the initialization parsing with the keyword `var`, so the changed name seems a little bit confusion to me :)